src/NuGet.Core/NuGet.Frameworks/FrameworkConstants.cs

```diff
@@ -186,6 +187,7 @@ public static class CommonFrameworks
             // .NET 5.0 and later has NetCoreApp identifier
             public static readonly NuGetFramework Net50 = new NuGetFramework(FrameworkIdentifiers.NetCoreApp, Version5);
             public static readonly NuGetFramework Net60 = new NuGetFramework(FrameworkIdentifiers.NetCoreApp, Version6);
+            public static readonly NuGetFramework Net70 = new NuGetFramework(FrameworkIdentifiers.NetCoreApp, Version7);
```

Comment:
You can always just not add this change as it's not really anything that requires your change. this is merely an optimization for restore.